dashboard/lib/src/rpc_model/base.dart

```diff
@@ -12,13 +13,20 @@ import 'package:meta/meta.dart';
 @immutable
 @internal
 abstract base mixin class Model {
-  @mustBeOverridden
+  static const _deepEquality = DeepCollectionEquality();
```

Comment:
do you want unordered given a sub-list could be [1,2,3] vs [2,1,3]?

My guess is no and that this is fine because _listEq in `build_status_response.dart` does ListEquality, requiring they be in-order.